### PR TITLE
SISRP-9408 - CC: Implement preferred name editing

### PR DESCRIPTION
**Pull request by christianv**
https://jira.berkeley.edu/browse/SISRP-9408

### CODE REVIEW

File: app/models/campus_solutions/person_name.rb [hunk 1]
```diff
@@ -44,9 +44,7 @@ def xml_filename
     def default_post_params
       super.merge(
         {
-          EFFDT: '27-JUL-15', # TODO fix hardcoding
-          EFF_STATUS: 'A',
-          COUNTRY_NM_FORMAT: '' # TODO ask Babu what this is
+          COUNTRY_NM_FORMAT: ''
```
Comment (christianv) [inline app/models/campus_solutions/person_name.rb]: FYI these changes were made possible by https://jira.berkeley.edu/browse/SISRP-9405 which was fixed today. /cc @ctweney

File: src/assets/javascripts/angular/services/profileService.js [hunk 2]
```diff
@@ -60,6 +62,29 @@ angular.module('calcentral.services').service('profileService', function() {
   };
 
   /**
+   * Find a certain item in with a specific code
```
Comment (paulkerschen) [inline src/assets/javascripts/angular/services/profileService.js]: @christianv Stray `in`?

**Comment by raydavis** [comment]
LGTM

File: src/assets/javascripts/angular/services/profileService.js (hunk 2 above)
Comment (christianv) [inline src/assets/javascripts/angular/services/profileService.js]: @paulkerschen thanks - should be fixed in a follow up commit

**Pull request merged**